packages/neuvector/generated-changes/overlay/questions.yaml

```diff
@@ -0,0 +1,224 @@
+questions:
+#image configurations
+- variable: registry
+  default: "docker.io"
+  description: image registry
+  type: string
+  label: Image Registry
+  group: "Container Images"
+- variable: controller.image.repository
+  default: "neuvector/controller.preview"
+  description: controller image repository
+  type: string
+  label: Controller image path
+  group: "Container Images"
+- variable: controller.image.tag
+  default: "5.0.0-preview.3"
+  description: image tag for controller
+  type: string
+  label: Controller Image Tag
+  group: "Container Images"
+- variable: manager.image.repository
+  default: "neuvector/manager.preview"
+  description: manager image repository
+  type: string
+  label: Manager image path
+  group: "Container Images"
+- variable:  manager.image.tag
+  default: "5.0.0-preview.3"
+  description: image tag for  manager
+  type: string
+  label: Manager Image Tag
+  group: "Container Images"
+- variable: enforcer.image.repository
+  default: "neuvector/enforcer.preview"
+  description: enforcer image repository
+  type: string
+  label: Enforcer image path
+  group: "Container Images"
+- variable: enforcer.image.tag
+  default: "5.0.0-preview.3"
```

Comment:
Preview 3 ?

Reply:
Somehow question.yaml is not matching the tag value from values.yaml. The good part is Rancher UI is showing tag value from values.yaml (5.0.0-b1) even though question.yaml has a different value(5.0.0-preview.3).

We will correct it to 5.0.0-b1

Reply:
Default may be overriden by values.yaml, you could just remove it to avoid confusion.